scripts/Create-Win32InstallerMUI.ps1

```diff
@@ -5,7 +5,8 @@
 
 param(
 	[Parameter(Mandatory=$true)]$packageName,
-	[Parameter(Mandatory=$true)]$version
+	[Parameter(Mandatory=$true)]$version,
+	[switch]$allLanguages
```

Comment:
Should probably be renamed to `allCultures` - but let's see whether it works in the first place ^^